index.html

```diff
@@ -301,6 +301,7 @@ <h2>
     interface DigitalCredential : Credential {
       readonly attribute DOMString protocol;
       [SameObject] readonly attribute object data;
+      static boolean clientAllowsProtocol(DOMString protocol);
```

Comment:
See rationale for change below...

```suggestion
      static boolean allowsProtocol(DOMString protocol);
```

Reply:
I still don't think we need to mention client here, even it it's to do with clients. The clients are an implementation detail.

Reply:
The clients play no part here (hence my objection to the inclusion of the client in the name); only the user agent. Why this method return immediately. I could live with:

* `DigitalCredential.userAgentAllowsProtocol()`